freqtrade/configuration/configuration.py

```diff
@@ -106,6 +111,8 @@ def load_config(self) -> Dict[str, Any]:
 
```

Comment:
Should `block_bad_exchanges` be moved to `exchange` ? it's really an exchange setting, not an experimental setting ... ?

Reply:
I guess no, it perfectly fits into experimental (that's for experiments with exchanges), plus leaving 'experimental' section alive with this setting will help fill it again with new experimental settings...

This setting can be left undocumented, probably...